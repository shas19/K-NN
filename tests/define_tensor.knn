let t = Tensor(4, [1.0, 2.0, 2.0, 1.0]) in 
let b = 5.0 in 
t