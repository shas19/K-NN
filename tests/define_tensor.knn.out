<T>
  <k>
    let t = tensor ( 4 , [ 0.10000000000000000e1 , 0.20000000000000000e1 , 0.20000000000000000e1 , 0.10000000000000000e1 ] ) in let b = 0.50000000000000000e1 in t ~> .
  </k>
  <env>
    .Map
  </env>
  <store>
    .Map
  </store>
  <nextLoc>
    0
  </nextLoc>
  <shape>
    .Map
  </shape>
</T>
